qutip/core/cy/_element.pyx

```diff
@@ -594,7 +594,7 @@ cdef class _ProdElement(_BaseElement):
         cdef double complex out = self._left.coeff(t) * self._right.coeff(t)
         return conj(out) if self._conj else out
 
-    cdef Data matmul_data_t(_ProdElement self, t, Data state, Data out=None):
+    cpdef Data matmul_data_t(_ProdElement self, t, Data state, Data out=None):
```

Comment:
What caused all the changes to `cpdef`?

Reply:
Sorry, I was multitasking.
This wasn't mean to be here.